ToDoApp/ToDoApp/CreateTaskView+ModelView.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  CreateTaskView+ModelView.swift
+//  ToDoApp
+//
+//  Created by Trong Hieu Vuong on 2022-12-02.
+//
+
+import Foundation
+import SwiftUI
```

Comment:
UI frameworks are not supposed to be imported in the business layer. ViewModel should be not aware of View